### ISSUE TITLE
[BUG] setting reconnectInterval to 0 does not disable reconnects

### ISSUE THREAD

**Issue by blueid-mathias-henze** [osiegmar/logback-gelf#12]
In `de.siegmar.logbackgelf.GelfTcpAppender`, the documentation for `reconnectInterval` says:

> Time interval (in seconds) after an existing connection is closed and re-opened.
> A value of 0 disables automatic reconnects. Default: 300 seconds.

but the implementation checks for < 0, so effectively reconnects on every message:
```
    private void connect() throws IOException {
        closeOut();

        final Socket socket = getSocket();
        outputStream = socket.getOutputStream();

        nextReconnect = reconnectInterval < 0
            ? Long.MAX_VALUE
            : System.currentTimeMillis() + (reconnectInterval * SEC_TO_MSEC);
    }
```

**Comment by osiegmar** [comment]
Thanks! This has already been fixed in the connection-pool branch which is supposed to be released shortly.

**Comment by osiegmar** [comment]
fixed in 1.1.0 which has just been released
